code/modules/projectiles/guns/ballistic/laser_gatling.dm

```diff
@@ -15,10 +15,10 @@
 	var/overheat_max = 40
 	var/heat_diffusion = 1
 
-/obj/item/weapon/minigunpack/New()
+/obj/item/weapon/minigunpack/Initialize()
```

Comment:
@kevinz000 New and Initialize aren't really that different. They are both called (eventually) when an object is created. If something is creating lag in that proc, this won't fix it.